### PR TITLE
OCLOMRS-924: Adds new member to an organisation.

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
# JIRA TICKET NAME:
[Add new member to an organization](https://issues.openmrs.org/browse/OCLOMRS-924)

# Summary:

# Caveats:
- I can't test it working yet, not sure if it's because of permissions, but I am reaching out to the backend people to help me.

### CODE REVIEW

**Comment by ibacher** [comment]
@hadijahkyampeire I've made some adjustments to this, but this works for me (I had to create a new organisation to get it to work). A couple of things I think we need to take account of:

1. The modal form doesn't disappear when it's submitted whether successfully or unsuccessfully.
2. In the case where we get a 403 error (or, indeed, another error) we should give the user some kind of feedback (maybe a notification in the notification bit?)

**Comment by hadijahkyampeire** [comment]
Thanks @ibacher , I am going to work on the notification and modal closing.

File: src/apps/organisations/redux/actions.ts [hunk 1]
```diff
@@ -122,13 +123,13 @@ const deleteOrganisationAction = createActionThunk(
...
-    GET_ORG_MEMBERS_ACTION,
-    api.organisation.retrieveMembers
+  GET_ORG_MEMBERS_ACTION,
+  api.organisation.retrieveMembers
 );
 
 const addOrgMemberAction = createActionThunk(
-  GET_ORG_MEMBERS_ACTION,
-  api.organisation.AddMember
+  CREATE_ORG_MEMBER_ACTION,
```
Comment (hadijahkyampeire) [inline src/apps/organisations/redux/actions.ts:131]: Sorry about this @ibacher , how did I miss that, it's my bad.

**Comment by hadijahkyampeire** [comment]
@ibacher , I catch the error message and I am displaying it in an Alert notification, but I couldn't test the successful adding, hope it works. Also, I close the dialog when loading is false, so that should cater for success or failure.

**Pull request merged**